src/cmd/package.go

```diff
@@ -336,6 +328,31 @@ func (o *packageMirrorResourcesOptions) run(cmd *cobra.Command, args []string) (
 		err = errors.Join(err, pkgLayout.Cleanup())
 	}()
 
+	// We don't yet know if the targets are internal or external
+	c, _ := cluster.New(ctx) //nolint:errcheck
+
+	if pkgConfig.InitOpts.RegistryInfo.Address == "" {
```

Comment:
We should check if the package has images before this validation, and check if the package has repos before the git server validation. 

For example, a user without an initialized cluster mirroring a package that doesn't include git repositories will get an error here if they only specify the `--registry-*` args.

Reply:
Agree that those would be useful flags

Reply:
I think addressing that gap is important enough to include as additional scope - WIP